sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/RemoteDependencyData.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using Azure.Core;
+using OpenTelemetry.Trace;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Models
+{
+    internal partial class RemoteDependencyData
+    {
+        // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md#connection-level-attributes
+        internal static readonly HashSet<string> SqlDbs = new HashSet<string>() { "mssql" };
+
+        public RemoteDependencyData(int version, Activity activity, ref TagEnumerationState monitorTags) : base(version)
+        {
+            string httpUrl = null;
+            string dependencyName;
+
+            if (monitorTags.activityType == PartBType.Http)
```

Comment:
minor: This can be rearranged now as we are no more calling the constructor. Not a blocker, I can create a separate PR later.